src/evaluate/inspect.py

```diff
@@ -31,41 +32,67 @@ class SplitsNotFoundError(ValueError):
     pass
 
 
-def list_metrics(with_community_metrics=True, with_details=False):
+def list_evaluation_modules(type=None, include_community=True, with_details=False):
     """List all the metrics script available on the Hugging Face Hub.
```

Comment:
Maybe you should clarify that this return a list of repo ids. The name tells me it returns a list of modules, which is a bit confusing since it really returns ids. Similar with the docstring

Reply:
I actually changed this a bit: for canonical modules the the namespace is omitted so this function returns what the user should put into `load()` function. Technically, `load("evaluate-metric/accuracy")` also works but I think we should use `load("accuracy")` as the standard.